test/expected/postgres/addresses.json

```diff
@@ -21,6 +14,12 @@
       "foreignKey": "customer_id",
       "foreignKeyName": "customerId",
       "as": "customer"
+    },{
+      "as": "city",
+      "foreignKey": "city",
+      "foreignKeyName": "cityKey",
+      "ref": "addresses",
+      "targetKey": "city"
```

Comment:
too much city so the test might work because of luck. Change at least the targetKey

Reply:
This will disapear as for now it is considered as a valid foreign key, but its a UNIQUE key, which I don't put in references (once I have pushed the logic in analyze-sequelize-table.js).

Reply:
It will not be counted as a reference once the logic is here :)